.prettierrc

```diff
@@ -1,15 +1,10 @@
+
```

Comment:
Did you intend to have this empty line?

Reply:
Not necessarily. But it doesn't do anything :P